dart/lib/src/protocol/max_response_body_size.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:meta/meta.dart';
+
+/// Describes the size of http response bodies which should be added to an event
+/// This enum might be removed at any time.
+@experimental
+enum MaxResponseBodySize {
```

Comment:
Since this is basically the same as `MaxRequestBody`, it probably makes sense to unify them as `MaxBodySize` or something along those lines.

Reply:
That makes a lot of sense but it'll be a breaking change, we can unify that on v7.